test/csuite/wt3184_dup_index_collator/main.c

```diff
@@ -44,10 +44,13 @@ compare_int(int32_t a, int32_t b)
 }
 
 static int32_t
-item_to_int(WT_ITEM *item)
+item_to_int(const WT_ITEM *item)
 {
+    int32_t buf;
```

Comment:
Can we pick a better name for this integer variable?  (`buf` sounds like a memory buffer).

Reply:
Done. Also added a comment explaining why memcpy is used instead of type cast.